pyface/i_heading_text.py

```diff
@@ -10,32 +10,109 @@
 
 """ Heading text. """
 
+import warnings
 
-from traits.api import HasTraits, Instance, Int, Interface, Str
-
-
-from pyface.i_image_resource import IImageResource
+from traits.api import HasTraits, Int, Interface, Str, observe
 
 
 class IHeadingText(Interface):
-    """ Heading text. """
+    """ A widget which shows heading text in a panel. """
 
     # 'IHeadingText' interface ---------------------------------------------
 
-    #: Heading level.
-    #
-    # fixme: Currently we ignore anything but one, but in future we could
-    # have different visualizations based on the level.
+    #: Heading level.  This is currently unused.
     level = Int(1)
 
     #: The heading text.
     text = Str("Default")
 
-    #: The background image.
-    image = Instance(IImageResource)
-
 
 class MHeadingText(HasTraits):
     """ The mixin class that contains common code for toolkit specific
     implementations of the IHeadingText interface.
     """
+
+    # 'IHeadingText' interface ---------------------------------------------
+
+    #: Heading level.  This is currently unused.
+    level = Int(1)
+
+    #: The heading text.
+    text = Str("Default")
+
+    def __init__(self, parent=None, **traits):
+        """ Creates the heading text. """
+
+        if "image" in traits:
+            warnings.warn(
+                "background images are no-longer supported for Wx and the "
+                "'image' trait will be removed in a future Pyface update",
+                PendingDeprecationWarning,
+            )
```

Comment:
can you create an issue for this deprecation - and put it in the 8.0.0 milestone?

Reply:
Will do.

Reply:
Done #978